### PR TITLE
Add Table, ShortAddress, AddressIcon, Metric, Label, Card

### PR DESCRIPTION
**Pull request by Hahahayabanan**

### CODE REVIEW

File: packages/components/src/icons/InfoIconV2.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import SvgIcon from '@material-ui/core/SvgIcon';
+
+function InfoIcon(props: React.ComponentProps<typeof SvgIcon>) {
```
Comment (in19farkt) [inline packages/components/src/icons/InfoIconV2.tsx:4]: rename file to InfoIcon

File: packages/components/src/Table/components/RowExpander/icons/Arrow.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+export const Arrow: React.FC = () => {
```
Comment (in19farkt) [inline packages/components/src/Table/components/RowExpander/icons/Arrow.tsx:3]: Move to icons

File: packages/components/package.json [hunk 3]
```diff
@@ -36,7 +36,15 @@
...
-    "classnames": "^2.0.0"
+    "@types/color": "^3.0.1",
+    "@types/mersenne-twister": "^1.1.2",
+    "@types/react-copy-to-clipboard": "^4.3.0",
+    "classnames": "^2.0.0",
+    "@types/ramda": "^0.27.14",
+    "ramda": "^0.27.1",
+    "mersenne-twister": "^1.1.0",
+    "color": "^3.1.2",
+    "react-copy-to-clipboard": "^5.0.2"
```
Comment (kinda-neat) [inline packages/components/package.json:47]: As I understand, these are peerDependencies for this package and dev dependencies for workspace-root. It's done in this way to be able to develop the package, and use host-project dependencies through peerDependencies, or maybe some of them really unique to the components package, we need to investigate this

File: packages/components/src/temp23/InheritBackgroundHackProvider.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useContext } from 'react';
+import { useTheme } from '@akropolis-web/styles';
+
+const InheritBackgroundHackContext = createContext<string | null>(null);
```
Comment (kinda-neat) [inline packages/components/src/temp23/InheritBackgroundHackProvider.tsx:4]: You don't need this, update @akropolis-web/styles and use it from there

File: packages/components/src/Label/Label.tsx [hunk 5]
```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import cn from 'classnames';
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+import { makeStyles } from '@akropolis-web/styles';
+
+import { ComingSoon } from '../ComingSoon/ComingSoon';
+import { InfoIconV2 } from '../icons';
```
Comment (kinda-neat) [inline packages/components/src/Label/Label.tsx:8]: forget to rename InfoIconV2 -> InfoIcon in `../icons`

**Pull request merged**